docs/pages/faq.rst

```diff
@@ -0,0 +1,51 @@
+Frequently Asked Question
+=========================
+
+
+Incorrect encoding
+------------------
+
+In Query protocol, Minecraft uses ISO 8859-1 for encoding all text (like MOTD,
+server name, etc.). This can cause problems with non-latin characters. To fix
+such error, you can re-encode text into UTF-8.
+
+.. code-block:: python
+
+    >>> query = JavaServer.lookup("my-server-ip.com").query()
+    >>> query.motd
+    'Ð\x9fÑ\x80Ð¸Ð²Ñ\x96Ñ\x82!'
+    >>> query.motd.encode("iso-8859-1").decode("utf-8")
+    'Привіт!'
+
+:attr:`query.motd <mcstatus.querier.QueryResponse.motd>` here can be anything,
+that contain incorrect encoding.
+
+
+How to get server image?
```

Comment:
Mention that you can also just paste the entire base64 image as a string result into the browser (most modern browsers will support this)

Reply:
Mentioned in [last force-push](https://github.com/py-mine/mcstatus/compare/9086ff3d9484d8cde97d4439741329a1a487017d..f71fea6b2261db29714c612dcf737c7c75465482), thanks!